yo1k/tic_tac_toe/test/core/test_game.py

```diff
@@ -214,7 +210,7 @@ def test_win(self) -> None:
                 None,
                 Action.new_occupy(Cell(0, 1))])
         logic = Logic((act_queue_px, act_queue_po))
-        for _ in range(0, len(act_queue_px.actions) + len(act_queue_po.actions)):
+        while len(act_queue_px.actions) + len(act_queue_po.actions) != 0:
```

Comment:
While `!= 0` is equivalent to `> 0` in this particular case, the latter conveys more info: while we are iterating the left side is non-negative and we are going to iterate until it reaches 0. `!= 0` does not convey the part _"while we are iterating the left side is non-negative"_. One can derive this part by actually looking at what we have in the left side, but that requires giving it a little bit more thought.

Reply:
Done.